flocker/control/_config.py

```diff
@@ -210,8 +210,18 @@ def convert(self):
         if volume:
             config['volume'] = volume
         config['restart_policy'] = self.convert_restart_policy()
+        cpu_shares = self.convert_cpu_shares()
```

Comment:
Doing this change in `_config.py` means it impacts our configuration language... but there's no test coverage from that direction. Given we're likely going to kill our configuration language I am of two minds about whether we should be updating it or making this change in `httpapi.py`. Most of the code is going to be thrown out (other than Fig) and so anything used by HTTP API will end up being moved to `httpapi.py` eventually anyway.

I already have to munge result for volumes (where JSON schema doesn't quite match config format) so I'd say do this in `httpapi.py`. And eventually we'll merge into one nice function.